includes/Checker/Checks/Plugin_Readme_Check.php

```diff
@@ -225,7 +225,7 @@ private function check_default_text( Check_Result $result, string $readme_file,
 	private function check_license( Check_Result $result, string $readme_file, Parser $parser ) {
 		$license          = $parser->license;
 		$matches_license  = array();
-		$plugin_main_file = WP_PLUGIN_DIR . '/' . $result->plugin()->basename();
+		$plugin_main_file = $result->plugin()->main_file();
```

Comment:
Similar update should be done in following files also.
- Plugin_Header_Text_Domain_Check
- Plugin_Readme_Check
- Plugin_Updater_Check
- Trademarks_Check

Reply:
Yes, you're right. I'll check it out